tests/Doctrine/Migrations/Tests/OutputWriterTest.php

```diff
@@ -19,7 +19,9 @@ public function testDefaultCallback() : void
     {
         $outputWriter = new OutputWriter();
 
-        self::assertNull($outputWriter->write('test message'));
+        $outputWriter->write('test message');
+
+        self::markTestIncomplete('Does not perform any real assertion.');
```

Comment:
`$this->addToAssertionCount(1)`, ref: https://github.com/sebastianbergmann/phpunit/pull/3348

Reply:
Nevermind, outdated